types/base.d.ts

```diff
@@ -1,10 +1,9 @@
 import { ClientAPI } from "./client";
-import { IterationAPI } from "./iteration";
```

Comment:
Why did iteration disappear?

Reply:
It existed in a first draft but was later included in the client API. This is supposed to be deleted long ago

Reply:
Maybe delete its types as well? I think I saw some.

Reply:
Ah, nope, sorry, I didn't explained it well. The Iteration interface and entity still exists, but the IterationAPI does not, since it exists now inside the clientAPI.

Reply:
Ah, ofc. My bad